acceptance/log/log.go

```diff
@@ -21,34 +21,76 @@ import (
 	"context"
 	"testing"
 
+	"sigs.k8s.io/kind/pkg/log"
+
 	"github.com/enterprise-contract/ec-cli/acceptance/testenv"
 )
 
 type Logger interface {
-	Log(args ...interface{})
-	Logf(format string, args ...interface{})
-	Printf(format string, v ...interface{})
+	Log(args ...any)
+	Logf(format string, args ...any)
+	Printf(format string, v ...any)
+	Warn(message string)
+	Warnf(format string, args ...any)
+	Error(message string)
+	Errorf(format string, args ...any)
+	V(level log.Level) log.InfoLogger
+	Info(message string)
+	Infof(format string, args ...any)
+	Enabled() bool
 }
 
 type logger struct {
 	t *testing.T
 }
 
 // Log logs given arguments
-func (l logger) Log(args ...interface{}) {
+func (l logger) Log(args ...any) {
 	l.t.Log(args...)
 }
 
 // Logf logs using given format and specified arguments
-func (l logger) Logf(format string, args ...interface{}) {
+func (l logger) Logf(format string, args ...any) {
 	l.t.Logf(format, args...)
 }
 
 // Printf logs using given format and specified arguments
-func (l logger) Printf(format string, args ...interface{}) {
+func (l logger) Printf(format string, args ...any) {
 	l.Logf(format, args...)
 }
 
+func (l logger) Warn(message string) {
+	l.Logf("[WARN ] %s", message)
+}
+
+func (l logger) Warnf(format string, args ...interface{}) {
```

Comment:
Is there a reason to use `interface{}` here instead of being consistent with the above code and using `any`? I understand that `any` is an alias for `interface{}` so I'm just wondering if I'm missing some subtle reason?

Reply:
Copy pasta